flash/core/data/process.py

```diff
@@ -344,14 +344,20 @@ def default_transforms() -> Optional[Dict[str, Callable]]:
 
     def pre_tensor_transform(self, sample: Any) -> Any:
         """Transforms to apply on a single object."""
+        if isinstance(sample, list):
+            return [self.current_transform(s) for s in sample]
         return self.current_transform(sample)
```

Comment:
Can we move this out somewhere since it's repeated in three places?

Reply:
Done !